### PR TITLE
Add ALP encrypted images

### PR DESCRIPTION
**Pull request by jlausuch**

### CODE REVIEW

**Comment by foursixnine** [comment]
I'm adding not ready tag for now until https://github.com/os-autoinst/openqa-trigger-from-obs/pull/182 is done

**Comment by jlausuch** [comment]
> I'm adding not ready tag for now until [os-autoinst/openqa-trigger-from-obs#182](https://github.com/os-autoinst/openqa-trigger-from-obs/pull/182) is done

This is independent. This can be merge already, it won't break anything.

**Review by foursixnine — APPROVED** [review]

File: job_groups/alp.yaml [hunk 1]
```diff
@@ -48,6 +48,10 @@ products:
     distri: alp
     flavor: kvm
     version: '0.1'
+  alp-0.1-kvm_encrypted-x86_64:
```
Comment (jknphy) [inline job_groups/alp.yaml:51]: now they are called 0.0.1

Comment (jlausuch) [reply job_groups/alp.yaml:51]: Damn, right!

**Pull request merged**